core/types/index.d.ts

```diff
@@ -1,5 +1,7 @@
 import { PassThrough } from 'stream';
 
+import { HttpError } from 'http-errors';
```

Comment:
We cannot do this, `'http-errors'` is a dependency which is no defined in the project. I'd rather copy the type to this file

like 

```
    interface HttpError extends Error {
        status: number;
        statusCode: number;
        expose: boolean;
        headers?: {
            [key: string]: string;
        };
        [key: string]: any;
    }
```

and we can remove it when commons-api and types lives in the same repo in the future. It is a workaround but I guess is valid for now.

Reply:
Or just add `"@types/http-errors"` as denpendency

Reply:
Sure
Sorry, I made these changes via web-editor